packages/Adapters/ClassicDTK/test/field/tstFieldTools.cpp

```diff
@@ -344,7 +344,7 @@ TEUCHOS_UNIT_TEST( FieldTools, scalar_field_test )
     }
 
     Teuchos::Array<double> q_norms;
-    int q = std::floor( ( 10 * std::rand() ) / RAND_MAX ) + 1;
+    int q = std::floor( ( 10 * (double)std::rand() ) / RAND_MAX ) + 1;
```

Comment:
I copied what was done for the other `rands`.  I think it's better to keep it uniform in the same file.